ci/pipelines/shared/jinja.variables.yml

```diff
@@ -122,7 +122,7 @@ tests:
   CALL_STACK_TIMEOUT: '7200'
   CPUS: '96'
   DUNIT_PARALLEL_FORKS: '24'
-  EXECUTE_TEST_TIMEOUT: 2h15m
+  EXECUTE_TEST_TIMEOUT: 6h
```

Comment:
please also increase the call stack timeout.  it should always be set 15 minutes less than the new timeout, e.g. 20700